packages/admin/admin/src/form/FinalFormChooseFile.tsx

```diff
@@ -0,0 +1,269 @@
+        onDrop,
+        accept,
+        disabled: disabled || (maxFiles && fieldValue.length === maxFiles),
+        multiple: multipleFiles,
+        maxSize: maxSize,
+        maxFiles,
+    });
+
+    // list of the accepted files
+    let files: File[];
+
+    if (Array.isArray(fieldValue)) {
+        files = fieldValue;
+    } else if (fieldValue.name !== undefined) {
+        files = [fieldValue];
+    } else {
+        files = [];
+    }
+
+    const rejectedFiles = fileRejections.map((rejectedFile) => (
+        <div key={rejectedFile.file.name} className={classes.rejectedFileListItem}>
+            <div className={classes.fileListText}>{rejectedFile.file.name}</div>
+            <IconButton color="error">{infoIcon}</IconButton>
+        </div>
+    ));
+
+    return (
+        <div className={classes.root}>
+            <div {...getRootProps()} className={classes.dropzone}>
+                <input {...getInputProps()} />
+                {!disableDropzone && (
+                    <div
+                        className={clsx(
+                            classes.droppableArea,
+                            disabled && classes.droppableAreaIsDisabled,
+                            isDragReject && classes.droppableAreaHasError,
+                        )}
+                    >
+                        {isDragReject && <div className={classes.droppableAreaError}>{infoIcon}</div>}
+                        <div className={classes.droppableAreaCaption}>
+                            <FormattedMessage id="comet.finalFormChooseFile.dropfiles" defaultMessage="Drop files here to upload" />
+                        </div>
+                    </div>
+                )}
+                {!disableSelectFileButton && (
+                    <Button disabled={disabled} variant="contained" color="primary" startIcon={selectIcon}>
+                        <FormattedMessage id="comet.finalFormChooseFile.selectfile" defaultMessage="Select file" />
+                    </Button>
+                )}
+            </div>
+            {files.length > 0 && (
+                <div className={classes.fileList}>
+                    {files.map((file) => (
+                        <div key={file.name} className={classes.fileListItem}>
+                            <div className={classes.fileListText}>{file.name}</div>
+                            <div className={classes.fileListItemInfos}>
+                                <Chip label={<PrettyBytes value={file.size} />} />
+                                <IconButton onClick={removeFile(file)}>{deleteIcon}</IconButton>
+                            </div>
+                        </div>
+                    ))}
+                </div>
+            )}
+            {fileRejections.length > 0 && <div className={classes.fileList}>{rejectedFiles}</div>}
+            {(fileRejections.length > 0 || isDragReject) && (
+                <div className={classes.errorMessage}>
+                    <Info color="error" />
```

Comment:
This should probably use the `infoIcon` variable and the default value of `infoIcon` should include the color. 
If the `infoIcon` also needs to be used without the color, this should probably be a separate icon e.g. `errorIcon`.